commands/bmc.vegman

```diff
@@ -363,24 +363,34 @@ function cmd_remoteimage {
 # @sudo cmd_remoteimage_connect admin
 # @doc cmd_remoteimage_connect
 # Connect to the remote server via the specified protocol
-#   --nbd <0..10>     use NBD protocol to connect the remote server with specified NBD device number
-#   address[:port]    remote server address
+#   --nbd <0..10>         use NBD protocol to connect the remote server with specified NBD device number
+#   -e,--export <name>    the export name from the NBD server
```

Comment:
Just call it as they do in NBD, "name of the export", not "export name".

Reply:
Done